setup.py

```diff
@@ -13,7 +13,7 @@ def read(fname):
 
 setup(
     name='pytest-leaks',
-    version='0.2.2',
+    version='0.3.dev',
```

Comment:
Pauli - please feel free to add yourself as co-author.  Your contribution deserves recognition!